ffi/callbacks/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM google/dart:2.7
```

Comment:
Can we use the official dart-sdk image instead?

Docker image info: https://hub.docker.com/_/dart